src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisShardConsumer.java

```diff
@@ -373,9 +382,9 @@ public boolean isSkipShardSyncAtWorkerInitializationIfLeasesExist() {
         return skipShardSyncAtWorkerInitializationIfLeasesExist;
     }
 
-    private enum TaskOutcome {
+    /*public enum TaskOutcome {
```

Comment:
Since you have moved this to IShardConsumer, why not just remove this?

Reply:
Just a carry over from testing. I will remove them.